alternator/executor.cc

```diff
@@ -2783,7 +2783,7 @@ static void verify_all_are_used(const rjson::value* field,
         return;
     }
     for (auto it = field->MemberBegin(); it != field->MemberEnd(); ++it) {
-        if (!used.contains(it->name.GetString())) {
+        if (!used.contains(rjson::to_string(it->name))) {
```

Comment:
Again, I'm confused.
Obviously used.contains(it->name.GetString()) worked, and didn't need to make a copy of the string. So why do we suddenly need to make a copy with `to_string`? If you use `to_string_view()` here - it doesn't work?

Or maybe even in the old code, even though we used `char*` it was implicitly converted into an std::string, so there is no performance regression here.

Reply:
For consistency we shouldn't use GetString directly I think. And there is this \0 handling. I think to construct string like that: string(chars, 7) is faster than string(chars) which needs to find \0 char.

Reply:
I see. I was hoping that if we used here to_string_view(), then even if today C++ doesn't know (?) how to search in the map without creating a std::string, in the future maybe it will be implemented. By forcing the call to to_string() to really create the std::string, you're giving up on this future optimization opportunity.
But this is a minor point (if at all a point). Never mind.

Reply:
Yes, I know it doesn't work "as expected" on string_view (without any copying), but because the unpatched version used "char*" and it worked - I thought perhaps by implicitly creating a std::string from it - I was hoping it can also work the same way for std::string_view. From this discussion I guess it can't - but I didn't check.